app/src/main/java/com/dsvag/currencyexchanger/data/models/latest/Coin.kt

```diff
@@ -2,12 +2,14 @@ package com.dsvag.currencyexchanger.data.models.latest
 
 import androidx.room.Entity
 import androidx.room.PrimaryKey
+import androidx.room.TypeConverters
+import com.dsvag.currencyexchanger.data.utils.Converters
 import com.google.gson.annotations.SerializedName
 
-@Entity
+@Entity(tableName = "coin_data")
```

Comment:
А чего просто `coin` не назвал?
```suggestion
@Entity(tableName = "coin")
```